components/tab/theme.css

```diff
@@ -0,0 +1,56 @@
+@import '@teamleader/ui-colors';
+
+.title-tab {
+  color: var(--color-neutral-darkest);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  outline: 0;
+  user-select: none;
+  position: relative;
+
+  &:hover {
+    background-color: rgba(130, 130, 140, 0.24);
+    cursor: pointer;
+  }
+
+  &:focus {
+    background-color: rgba(130, 130, 140, 0.12);
+    box-shadow: 0 0 0 2px rgba(130, 130, 140, 0.24);
+  }
+
+  &::after {
+    content: '';
+    position: absolute;
+    display: block;
+    color: var(--color-teal-darkest);
+    box-shadow: inset 0px -3px 0px 0px var(--color-mint);
+    height: 3px;
+    width: 100%;
+    bottom: 0;
+    transition: transform .25s ease-in-out;
```

Comment:
can you use the transition variables from `@teamleader/ui-animations`?

 :ok: by @lowiebenoot on 29/11/2017 15:46:59